scripts/check-codeowners.sh

```diff
@@ -1,23 +1,27 @@
 #!/usr/bin/env bash
 
 cat <<EOF
-# See https://github.com/scottrigby/prometheus-helm-charts/issues/12
+# See https://github.com/prometheus-community/helm-charts/issues/12
 # https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 # https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
 # The repo admins team will be the default owners for everything in the repo.
 # Unless a later match takes precedence, they will be requested for review when someone opens a pull request.
 * @prometheus-community/helm-charts-admins
 
-/.github/workflows/ @prometheus-community/helm-charts-admins @jkroepke @GMartinez-Sisti
-/renovate.json @prometheus-community/helm-charts-admins @jkroepke @GMartinez-Sisti
+/.github/workflows/ @prometheus-community/helm-charts-admins @GMartinez-Sisti
+/renovate.json @prometheus-community/helm-charts-admins @GMartinez-Sisti
 
 EOF
 
-for DIR in $(ls -1 -d ./charts/*)
-do
-  FILE="$DIR/Chart.yaml"
-  DIR=$(echo $DIR | sed 's/^\.//')
-  MAINTAINERS=$(yq e '.maintainers.[].url' $FILE | sed 's!^https://github.com/!@!' | sort --ignore-case)
-  echo $DIR/ $MAINTAINERS
+get_maintainers() {
+  local chart_file="$1"
+  yq e '.maintainers.[].url' "${chart_file}" |
+    sed 's!^https://github.com/!@!' |
+    sort --ignore-case
```

Comment:
I don't know your system, but sort depends on the system local and MacOS do some [shenanigans](https://unix.stackexchange.com/questions/362728/why-does-gnu-sort-sort-differently-on-my-osx-machine-and-linux-machine) as well.

```suggestion
    LC_ALL=C sort --ignore-case
```